packages/story-editor/src/components/panels/shared/hotlinkModal.js

```diff
@@ -125,6 +163,12 @@ function HotlinkModal({
       const hotlinkInfo = await getHotlinkInfo(link);
       const shouldProxy = await checkResourceAccess(link);
 
+      if (shouldProxy && !canUseProxy) {
```

Comment:
In my PR, I do something similar. But instead of returning early, I throw a error. See https://github.com/GoogleForCreators/web-stories-wp/pull/11510/files#diff-63f158a92c1b323c663fb4b9fcf8a3d81ed7097c0dde851f6bec2a4d6812631fR90

Could we do something similar.

Reply:
The difference is that throwing an error like in your PR would mean calling `onError()` (and thus `trackError()`), which I don't want for this case, which is why this only sets the message but nothing else.

Reply:
Right, that is a good point. Might be worth a comment, that not tracking the error is by design here.